MaintenanceApi.yaml

```diff
@@ -425,7 +427,129 @@ paths:
       security:
       - MaintenanceDefinitionAuth:
         - fraihmwork.maintenance.write
-                
+  /maintenance/unscheduled:
```

Comment:
Is there a reason that *everything* here is under a maintenance path? The spec already says everything is under api/maintenance/v0?

Reply:
This will be handled in future issue